CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 ## 1.1.1 (Unreleased)
+
+NOTES:
+
+* The provider is now aligned with Terraform 0.12 SDK which is required for Terraform 0.12 support. This version of terraform is more strict with syntax enforcement. If you old configuration doesn't work post upgrade, please verify syntax against the updated documentation.
```

Comment:
Hi Anna, Is the updated documentation you refer to our documentation or Terraform documentation? Can you please make it explicit. Rest look great!